assert_test.go

```diff
@@ -0,0 +1,84 @@
+package taskflow_test
+
+import (
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func assertTrue(t testing.TB, got bool, msg string) {
+	if got {
+		return
+	}
+	t.Helper()
+	t.Errorf("%s\ngot: [%v], want: [true]", msg, got)
+}
+
+func assertContains(t testing.TB, got string, want string, msg string) {
```

Comment:
This function declartion is inconsistent comparing to other assertion functions. In all other functions `want` argument is before `got`.
Personally I would change all other declarations (not this one), because at stated in [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures):

>Note that the order here is actual != expected, and the message uses that order too. Some test frameworks encourage writing these backwards: 0 != x, "expected 0, got x", and so on. Go does not.

Let me know if you have time to refactor it this weekend. If not then I can do it myself 😉

Reply:
Addressed https://github.com/pellared/taskflow/pull/36/commits/2f78b4dc6001cc78d9fb28e967a06cca26211219

Reply:
Thank you